lib/temp.js

```diff
@@ -42,6 +42,12 @@ var parseAffixes = function(rawAffixes, defaultPrefix) {
   return affixes;
 };
 
+var getMode = function(rawAffixes) {
+  if ((rawAffixes == null) || (rawAffixes.mode == null)) {
+    return 0600;
+  }
+  return parseInt(rawAffixes.mode) || 0600;
```

Comment:
Possibly, `getMode()` should throw and error if `rawAffixes` is not an integer rather than silently failing here.

Although, if this library has a precedence of not throwing on parameter validation then ignore me.